type.go

```diff
@@ -27,6 +27,14 @@ type Queue[T constraints.Ordered] interface {
 	// and return the index if value matches with value
 	// in the queue
 	FindIndex(value T) (int, bool)
+
+	// Serobot is ingrained in our culture. Why queue when you
+	// can just jump the queue?
+	Serobot(value T)
```

Comment:
Brooooo this is illegal in our country XDDD, we can't just **serobot** it if we don't have any **money** here XD XD XD